packages/web-app-files/src/components/SideBar/Shares/Links/ListItem.vue

```diff
@@ -31,22 +31,23 @@
           v-if="linkShare.hasPassword"
           v-oc-tooltip="$gettext('This link is password-protected')"
           name="lock-password"
-          class="oc-files-file-link-has-password"
+          class="oc-files-file-link-has-password oc-mr-xs"
```

Comment:
I think I would make sense to have a ~xtra-small~ small margin here to have even spacing between the icons
![screenshot_003159](https://github.com/user-attachments/assets/d773f657-a0aa-447c-8db6-c3e1a032be16)

Reply:
I'm pretty sure it's even 🤔  All elements have `xs` margin, resulting in a total margin of `s` between the icons. It appears to be different in that case because the 3-dots-icon is so narrow compared to the other icons.

Reply:
ah , that thing, you are right 🙈